### DISCUSSION TITLE
Is there any way to import revive.toml?

### DISCUSSION THREAD

**Discussion by yuonoda** [golangci/golangci-lint#2111] [Q&A] [answered]
Hi there, 

I've used this golangci-lint on Github Actions to automate lint testing on my repository.
However, there is an independent revive.toml file in the repo to run lint locally with revive.
So I'd like to make golangcil-lint import this revive.yaml to lint my code even on Github Actions.
Is this possible?

**Comment by ldez** [comment, marked as answer]
Hello,

you just have to convert your `revive.toml` file to YAML and add those data in the `revive` section of the golangci-lint configuration.

- to convert TOML to YAML, you can find online tools, ex: https://www.convertsimple.com/convert-toml-to-yaml/
- the  `revive` section of the golangci-lint configuration: https://golangci-lint.run/usage/linters/#revive

It's not possible to read an external `revive.toml` file from golangci-lint.

**Reply by yuonoda** [reply]
Hi Idez,

Thank you for your rapid and kind reply! 
Okay, I'm going to convert my TOML into YAML on revive section.
Thank you!
